src/MobxStorePersist.ts

```diff
@@ -0,0 +1,118 @@
+import { action, IReactionDisposer, makeObservable, observable, reaction, runInAction, toJS } from 'mobx';
+import { StorageConfiguration } from './StorageConfiguration';
+import { PersistenceDecoratorOptions } from './types';
```

Comment:
This should probably be renamed:
```suggestion
import { PersistenceOptions } from './types';
```